gemet/thesaurus/management/commands/check_spreadsheet.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+from gemet.thesaurus.models import Property
+
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+import re
+
+
+LABELS = {
+    'RT (GEMET)': True,
+    'RT (new)': False,
+    'BT (GEMET)': True,
+    'BT (new)': False,
+    'NT (GEMET)': True,
+    'NT (new)': False,
+}
+
+
+def check_term_database_existence(term):
+    return Property.objects.filter(value=term).exists()
+
+
+def split_text_into_terms(raw_text):
+    pattern = re.compile("[^a-zA-Z\d \-\\)\\(:]")
+    term_list = pattern.split(raw_text)
+    term_list = map(lambda term: term.strip(' \t\n;,.:').lower(), term_list)
+    term_list = filter(lambda term: term != '', term_list)
```

Comment:
Using both `map` and `filter` means you iterate over the list twice. A list comprehension, or simply using a for loop to create the list would be better. Also, please keep in mind that *usually* a list comprehension is [easier to read](http://stackoverflow.com/questions/1247486/python-list-comprehension-vs-map) than `map` or `filter` and also Guido wanted to [drop them](http://www.artima.com/weblogs/viewpost.jsp?thread=98196) from Python3.